wagtail/admin/tests/pages/test_create_page.py

```diff
@@ -450,13 +450,9 @@ def test_create_simplepage_post_submit(self):
         self.assertFalse(page.live)
         self.assertFalse(page.first_published_at)
 
-        # The latest revision for the page should now be in moderation
-        self.assertTrue(page.get_latest_revision().submitted_for_moderation)
+        # The page should now be in moderation
+        self.assertEqual(page.current_workflow_state.status, page.current_workflow_state.STATUS_IN_PROGRESS)
 
-        # Check that the moderator got an email
-        self.assertEqual(len(mail.outbox), 1)
-        self.assertEqual(mail.outbox[0].to, ['moderator@email.com'])
-        self.assertEqual(mail.outbox[0].subject, 'The page "New page!" has been submitted for moderation')
```

Comment:
What's the rationale for removing this? (If the old implementation sent out an email at this point whereas the new one doesn't, isn't that an issue we need to address?)

Reply:
The new one does, but this is tested more thoroughly in `test_workflows`